step.yml

```
@@ -1,7 +1,39 @@
 title: Manage iOS Code Signing
-summary: TODO
+summary: Automatically manages your code signing files.
 description: |-
-  TODO
+  The **Manage iOS Code Signing** Step takes care of automatically code signing your project before it's built on Bitrise.
```

Technically this step is not code signing anything, it just prepares the required files for a successful code signing (which Xcode will do in a subsequent step)

```suggestion
  The **Manage iOS Code Signing** Step takes care of setting up the required code signing assets before the project is built on Bitrise.
```